libnopegl/src/path.c

```diff
@@ -185,6 +185,103 @@ int ngli_path_close(struct path *s)
     return 0;
 }
 
+static const char *strip_separators(const char *s)
+{
+    while (*s && strchr(" ,\t\r\n", *s)) /* comma + whitespaces */
+        s++;
+    return s;
+}
+
+static const char *load_coords(float *dst, const char *s, size_t n)
+{
+    for (size_t i = 0; i < n; i++) {
+        s = strip_separators(s);
+        dst[i] = strtof(s, (char **)&s);
```

Comment:
Are coords limited to float (and not double) ?

Reply:
Yeah doubles are reserved for time (and within animations but only because we want to be able to animate the time)

Reply:
I mean on the svg side + is strof() able to read the entire string and truncate appropriately  if needed to perform the conversion from something double to float  ?

Reply:
> Unless stated otherwise, numeric values in SVG attributes and in properties that are defined to have an effect on SVG elements must support at least all finite single-precision values supported by the host architecture. 

Source: https://www.w3.org/TR/SVG2/types.html#Precision

See also the first 2 paragraphs here: https://oreillymedia.github.io/Using_SVG/extras/ch08-precision.html

`strtof()` seems to return `HUGE_VALF` in our case if it overflows, whatever that is. I also don't think there is a limit to the length of the string if that's what you had in mind (also, AFAIK `double` don't necessarily have more digits in their strings)

Reply:
Thanks.